### DISCUSSION TITLE
YAML config option?

### DISCUSSION THREAD

**Discussion by ghost** [jpanther/congo#200] [Ideas]
Hi, I'm an enthusiastic YAML user and dedicated TOML hater. Does Congo offer a pre-made config.yml file? I've noticed some theme developers offer both. Thanks for your work.

**Comment by jpanther** [comment]
The short answer is, no. I don't have a particular preference for either format, and I selected the ones in use in the theme purely as that's what is used by Hugo. While I think the logic behind using TOML for config and YAML for front matter is flawed, it is what it is. There's plenty of converters out there that can translate the config for you.
